src/rest/validateHandler.go

```diff
@@ -0,0 +1,107 @@
+package rest
+
+import (
+	"devicemanager/config"
+	"devicemanager/rest/redfish"
+	"encoding/json"
+	"fmt"
+	"github.com/kataras/iris/v12"
+	"github.com/kataras/iris/v12/context"
+	"github.com/sirupsen/logrus"
+	"net/http"
+)
+
+type validateHandler struct {
+	cfg config.Config
+}
+
+func (v *validateHandler) handle(ctx iris.Context) {
+	if ctx.GetHeader("Authorization") == "" {
+		ctx.StatusCode(http.StatusUnauthorized)
+		logrus.Error("No valid authorization")
+		ctx.WriteString("No valid authorization")
+		return
```

Comment:
Below error messages are moved to a variable, as they are uses twice. Can we be consistent?